client/webserver/site/src/js/mm.ts

```diff
@@ -54,7 +56,8 @@ export default class MarketMakerPage extends BasePage {
     )
 
     Doc.bind(page.addBotBtn, 'click', () => { this.showAddBotForm() })
-    Doc.bind(page.startBotsBtn, 'click', () => { this.showPWSubmitForm() })
+    Doc.bind(page.addBotBtn2, 'click', () => { this.showAddBotForm() })
```

Comment:
Instead of `addBotBtn` and `addBotBtn2 `, maybe `addBotBtnNoExisting` `addBotBtnWithExisting`? Will be confusing which is which.